install-cloud-sdk-build-task/README.md

```diff
@@ -10,3 +10,27 @@ this task is unnecessary. If, however, the Google Cloud SDK is installed for
 the build agent user only, or not installed at all, you can use this task to find
 or install it. Use this task before calling any other tasks that require the Google
 Cloud SDK.
+
+## Parameters
+
+![Install Cloud SDK Parameters][InstallCloudSdkParameters]
+
+  - **Send anonymous usage statistics to Google**: Help make Google Cloud SDK better
+  by automatically sending [anonymous usage statistics][UsageStats]
+  to Google.
+  - **Config**: The name of the configuration to create or activate. If left blank, the
+  active config will be used.
+  - **Project**: Configure the Cloud SDK's current default project. If left blank, the
+  current configuration will not change.
+  - **Region**: Configure the Cloud SDK's current default compute/region. If left blank,
+  the current configuration will not change.
+  - **Zone**: Configure the Cloud SDK's current default compute/zone. If left blank,
+  the current configuration will not change.
+  - **Install Path**: The path on the agent to install the Cloud SDK. Defaults to '%LOCALAPPDATA%\\Google\\Cloud SDK'.
```

Comment:
Is there a line limit that you are trying to not exceed? If so, then I think you should split this into another line (I'm asking because it seems like you were trying to do that in the other file).